composer.json

```diff
@@ -20,7 +20,7 @@
     "GPL-2.0+"
   ],
   "require": {
-    "typo3/cms-core": ">=7.6.0,<8.9.99"
+    "typo3/cms": ">=7.6.13,<8.9.99"
```

Comment:
@NamelessCoder 
Here is my take on that: 
http://insight.helhum.io/post/155297666635/typo3-extension-dependencies-revisited